itou/siaes/models.py

```diff
@@ -444,6 +442,33 @@ def set_admin_role(self, active, user):
         self.updated_by = user
 
 
+class SiaeJobDescriptionQuerySet(models.QuerySet):
+    def with_job_applications_count(self, filters=None):
+        if filters:
+            filters = Q(**filters)
+
+        return self.annotate(
+            job_applications_count=Count(
+                "jobapplication",
+                filter=filters,
+            ),
+        )
+
+    def with_is_popular(self):
```

Comment:
J'expliquerais en commentaire que la méthode attache un attribut `is_popular`, ça aiderait à comprendre le nom de la méthode car le `is` est bizarre quand tu lis la requête en dehors :

`SiaeJobDescription.objects.with_is_popular()`

Reply:
C'est vrai. Je viens de la renommer en `with_annotation_is_popular`, c'est plus clair.